SSINS/sky_subtract.py

```diff
@@ -37,29 +37,49 @@ def read(self, filename, diff=False, flag_choice=None, INS=None, custom=None,
         Args:
             filename (str or list of str): The filepath(s) to read in.
             diff (bool): If True, and data was read in, then difference the visibilities in time
+            diff_freq (bool): If True, and data was read in, then difference the visibilities in freq
             flag_choice: Sets flags for the data array on read using apply_flags method.
             INS: An INS object for apply_flags()
             custom: A custom flag array for apply_flags()
             kwargs: Additional kwargs are passed to UVData.read()
+            override_keyword (str): sets a keyword to override to True regardless of internal diffs.
```

Comment:
Say which options are available and make a tutorial if you haven't.

Reply:
https://github.com/mwilensky768/SSINS/pull/91/commits/68fb0ba04fb2ac9dc132cd721b88d658c73f763f resolved this.